gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -1037,7 +1037,7 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
       type = gst_tensor_get_element_size (config->info.info[0].type);
 
       /** colorspace * width * height * type */
-      frame_size = color * width * height * type;
+      frame_size = (gsize) color *width * height * type;
```

Comment:
minor things.
```suggestion
      frame_size = (gsize) color * width * height * type;
```
L1068 also.

Reply:
For me, this was also a surprise. The indent-format system issued me a mistake and the following patch was proposed.
```
@@ -1037,7 +1037,7 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
       type = gst_tensor_get_element_size (config->info.info[0].type);
 
       /** colorspace * width * height * type */
-      frame_size = (gsize) color * width * height * type;
+      frame_size = (gsize) color *width * height * type;
 
       /** supposed 1 frame in buffer */
       g_assert ((buf_size / self->frame_size) == 1);
@@ -1065,7 +1065,7 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
         /**
          * Refer: https://gstreamer.freedesktop.org/documentation/design/mediatype-video-raw.html
          */
-        size = offset = (size_t) color * width * type;
+        size = offset = (size_t) color *width * type;
 
         g_assert (offset % 4); /** Internal logic error! */
         if (offset % 4) {

```
And I was forced to do so.

Reply:
It seems to be a problem with the LGTM system.
You should fix the system. :)
Please modify this regardless of the system.

Reply:
No, there is error from TAOS-Ci (indent-format).

Reply:
LGTM does not check the format code.

Reply:
Of course I can try to add a space. If you say you need!

Reply:
Oh.. You're right. TAOS-CI detected it.
Let's find the problem later.

Reply:
I checked it on my local PC. Thanks. Please revert your last change.

Reply:
Will I revert?

Reply:
Yes. please.

Reply:
One moment!

Reply:
Done!

Reply:
@tdrozdovsky How about this as temporal solution for weird indent.
```
frame_size = (gsize) (color * width * height * type);
```

Reply:
@gichan-jang  Done!